### DISCUSSION TITLE
How does one update dependencies?

### DISCUSSION THREAD

**Discussion by Thanaen** [oven-sh/bun#5149] [Q&A]
I've tried to update my project's dependencies with `bun update`, but I get the impression that it's waiting for us to list the dependencies to be updated. Is there a way to update all the dependencies, and indicate that you want to update them beyond the version indicated in package.json? (like `pnpm up -L`)

**Comment by melroy89** [comment]
There is no such flag as far as I know.. So what would be a new feature request for the issues.

Introducing some -L or `--latest` flag. 

At the same time also `bun outdated` isn't yet implemented either.
